rpc_jobs/rpc_openstack.yml

```diff
@@ -0,0 +1,97 @@
+- project:
+    name: "rpc-openstack"
+    repo_url: "https://github.com/rcbops/rpc-openstack"
+    branches:
+      - "master"
+    image:
+      - xenial:
+          IMAGE: "Ubuntu 16.04.2 LTS prepared for RPC deployment"
+    scenario:
+      - ironic:
+          TRIGGER_PR_PHRASE_ONLY: true
+      - "swift"
+    action:
+      - deploy:
+          FLAVOR: "performance2-15"
+          REGIONS: "DFW"
+          FALLBACK_REGIONS: "IAD"
+    jobs:
+      - 'PR_{name}-{image}-{scenario}-{action}'
+
+- project:
+    name: "rpc-openstack-newton"
+    repo_url: "https://github.com/rcbops/rpc-openstack"
+    branches:
+      - "newton.*"
+    image:
+      - trusty:
+          IMAGE: "Ubuntu 14.04.5 LTS prepared for RPC deployment"
+      - xenial:
+          IMAGE: "Ubuntu 16.04.2 LTS prepared for RPC deployment"
+    scenario:
+      - ironic:
+          TRIGGER_PR_PHRASE_ONLY: true
+      - "swift"
+    action:
+      - deploy:
+          FLAVOR: "performance2-15"
+          REGIONS: "DFW"
+          FALLBACK_REGIONS: "IAD"
+    jobs:
+      - 'PR_{name}-{image}-{scenario}-{action}'
+
+- project:
+    name: "rpc-openstack-mitaka"
+    repo_url: "https://github.com/rcbops/rpc-openstack"
+    branches:
+      - "mitaka.*"
+    image:
+      - trusty:
+          IMAGE: "Ubuntu 14.04.5 LTS prepared for RPC deployment"
+    scenario:
+      - "ceph"
+      - "swift"
+    action:
+      - deploy:
+          FLAVOR: "performance2-15"
+          REGIONS: "DFW"
+          FALLBACK_REGIONS: "IAD"
+    jobs:
+      - 'PR_{name}-{image}-{scenario}-{action}'
+
+- project:
+    name: "rpc-openstack-liberty"
+    repo_url: "https://github.com/rcbops/rpc-openstack"
+    branches:
+      - "liberty.*"
+    image:
+      - trusty:
+          IMAGE: "Ubuntu 14.04.5 LTS prepared for RPC deployment"
+    scenario:
+      - "ceph"
+      - "swift"
+    action:
+      - deploy:
+          FLAVOR: "performance2-15"
+          REGIONS: "DFW"
+          FALLBACK_REGIONS: "IAD"
+    jobs:
+      - 'PR_{name}-{image}-{scenario}-{action}'
+
+- project:
+    name: "rpc-openstack-kilo"
+    repo_url: "https://github.com/rcbops/rpc-openstack"
+    branches:
+      - "kilo.*"
+    image:
+      - trusty:
+          IMAGE: "Ubuntu 14.04.5 LTS prepared for RPC deployment"
+    scenario:
+      - "swift"
```

Comment:
I think ceph was excluded from kilo testing because of leapfrogs, but as leapfrog testing is handled elsewhere do we want to re-instate that here?

Reply:
@mattt416 Ceph was excluded from Kilo testing because kilo is only being tested for the purpose of leapfrogs, not for the purpose of releasing. If the leapfrog/RPC-O team wants to support Ceph upgrades then it can always add more tests later.